elife/multiservice.sls

```diff
@@ -0,0 +1,55 @@
+
+# systemd managed group processes
+
+# 1. N number of the given process are running
+# 2. the pool of processes can be grown or shrunk
+# 3. broken processes fail highstate as they ordinarily would
+
+{% for process, opts in pillar.elife.multiservice.services.items() %}
+    {% set num_processes = opts["num_processes"] %}
+
+# ensure the target state is available and running
+{{ process }}-controller.target:
+    file.managed:
+        - name: /lib/systemd/system/{{ process }}-controller.target
+        - source: salt://elife/templates/process-controller.target
+
+    service.running:
```

Comment:
this seems too early as at this point the right number of processes hasn't been enabled yet

Reply:
`systemctl restart tester-controller.target` works to restart all processes in parallel (the holy grail), will add it.